src/app/routes/admin/api/feedbacksHandler.ts

```diff
@@ -0,0 +1,47 @@
+//
+// LESERKRITIKK v2 (aka Reader Critics)
+// Copyright (C) 2017 DB Medialab/Aller Media AS, Oslo, Norway
+// https://github.com/dbmedialab/reader-critics/
+//
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the GNU General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option) any later
+// version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program. If not, see <http://www.gnu.org/licenses/>.
+//
+
+import {
+	Request,
+	Response,
+} from 'express';
+
+import {
+	errorResponse,
+	okResponse,
+} from 'app/routes/api/apiResponse';
+
+import { feedbackService } from 'app/services';
+
+/**
+ * Provides with whole list of existing feedbacks
+ * Not filtering, no page or limit query params are taken into account
+ */
+export function list (requ: Request, resp: Response) {
+	const notFound = 'Resourse not found';
+	feedbackService.getRange().then((fbacks) => {
+		if (fbacks.length > 0) {
```

Comment:
Just a note for the future, can we simplify this code even more? A check against result size will apply to pretty much all bulk queries. This if/else can be easily put into a function (which might belong into the `apiResponse` module) and then calling the service query function + delivering the response will become nice and short:
```
feedbackService.getRange()
.then(feedbacks => bulkResponse(resp, feedbacks))
.catch(error => errorResponse(resp, error));
```
Not for this branch, but probably later...

Also there should be a common function to extract skip, limit and sort parameters from the request data. These parameters are currently missing here and should be added soon. But now let's get this delivered.